### DISCUSSION TITLE
Modify the style of written content

### DISCUSSION THREAD

**Discussion by weironz** [GitbookIO/community#526] [Feature Requests]
![chrome_fzD8fgmrW6](https://github.com/GitbookIO/community/assets/33590311/e7652620-8120-408c-af2b-3dd0a6a01bdf)

I have written three lines of content, which could be 20 lines. 

I would like to select these three lines and convert them into non list, ordered lists, or code blocks at once. I think this is a very basic and useful feature. Can GitBook support it?

**Comment by thebiglabasky** [comment]
Hi
This is indeed something we would like to support but don't have any timelines on yet. Our efforts right now are focused on improving the published documentation experience which is taking significant effort. 
We'll update this conversation once we will get back on improvements to the editor.
